controllers/error.controller.js

```diff
@@ -10,8 +10,10 @@ exports.handleCustomPaths = (err, req, res, next) => {
 
 exports.handleSQLErrors = (err, req, res, next) => {
   if (err.code === "22P02" || err.code === "23502") {
+    console.log(err);
```

Comment:
we don't really need to console.log these errors only 500 errors because PSQL throws its own errors